tests/CodeSnifferRunner.php

```diff
@@ -16,6 +16,10 @@ public function __construct()
             define('PHP_CODESNIFFER_VERBOSITY', 0);
         }
 
+        if (!defined('PHP_CODESNIFFER_CBF')) {
```

Comment:
why not `defined('PHP_CODESNIFFER_CBF') or define('PHP_CODESNIFFER_CBF', false)`?